### PR TITLE
add working/resting feature

### PR DESCRIPTION
**Pull request by jvmaia**
enhancement #2

### CODE REVIEW

**Comment by ivoronin** [comment]
Please fix menu layout.

before:
image

after:
image

File: TomatoBar/TomatoBarController.swift [hunk 1]
```diff
@@ -69,8 +81,10 @@ public class TomatoBarController: NSViewController {
     override public func viewDidLoad() {
         super.viewDidLoad()
         /* Register defaults */
-        UserDefaults.standard.register(defaults: ["intervalLength": 25,
-                                                  "isSoundEnabled": true])
+        UserDefaults.standard.register(defaults: ["workingIntervalLength": 25,
+                                                  "isSoundEnabled": true,
+                                                  "restingIntervalLength": 5,
+                                                  "isResting": true])
```
Comment (ivoronin) [inline TomatoBar/TomatoBarController.swift]: isResting is an app state variable, not user setting, it shouldn't be registered here

Comment (jvmaia) [reply TomatoBar/TomatoBarController.swift]: thanks for the feedback, i'm really new to swift/macos development and i don't know how to do somethings

**Comment by Uffje** [comment]
Is this already available or is there a way to get this working?

**Comment by Abhi0725** [comment]
is this available now?

**Comment by Uffje** [comment]
> is this available now?

I ended up building jvmaia's, which works fine.

**Pull request closed without merging**